pkg/controller/backup_controller.go

```diff
@@ -527,6 +531,34 @@ func (c *backupController) runBackup(backup *pkgbackup.Request) error {
 		backup.Status.Phase = velerov1api.BackupPhaseCompleted
 	}
 
+	// Find and add the PodVolumeBackup after the backup is processed
```

Comment:
Instead of using the lister here to get the related pod volume backups, I think it would make more sense to modify https://github.com/heptio/velero/blob/master/pkg/restic/backupper.go#L39 to return a slice of PodVolumeBackups that were created, replacing the `map[string]string` that's currently being returned.  Then, the `item_backupper` can just grab them and assign them to `ib.backupRequest.PodVolumeBackups` here: https://github.com/heptio/velero/blob/master/pkg/backup/item_backupper.go#L222